lib/testsuite/cucumber.js

```diff
@@ -0,0 +1,109 @@
+const TestSuite =  require('./');
+const {Logger} = require('../utils');
+let Cli;
+let UserCodeRunner;
+try {
+  Cli =  require('@cucumber/cucumber/lib/cli').default;
+  UserCodeRunner = require('@cucumber/cucumber/lib/user_code_runner').default;
+} catch (err) {
+  if (err.code === 'MODULE_NOT_FOUND') {
+    Logger.error('Cucumber not installed');
```

Comment:
The way we've done this elsewhere is to create a new Error object and, a detailedErr property and then throw it. For instance [here](https://github.com/nightwatchjs/nightwatch/blob/v2/lib/api/protocol/_base-action.js#L21).